README.md

```diff
@@ -93,15 +93,19 @@ In order to build the language the following are needed:
 The `ref_impl` directory contains the reference implementation parser, type checker, interpreter, and command line runner. In this directory, build and test the Bosque reference implementation with:
 
 ```none
-npm install && npm run-script build && npm test
+npm install --production=false && npm run-script build && npm test
 ```
 
 ### Command Line Execution
 
-The `ref_impl` directory contains a simple command line runner for standalone Bosque (`.bsq`) files. These files must have a single `entrypoint` function called `main()` (see [some examples](ref_impl/src/test/apps)). The code in the file can be parsed, type checked, and executed with:
-
+The `ref_impl` directory contains a simple command line runner for standalone Bosque (`.bsq`) files. These files must have a single `entrypoint` function called `main()` (see [some examples](ref_impl/src/test/apps)). 
+Firstly you should compile `app_runner.ts`:
+```none
+tsc --downlevelIteration ref_impl/src/test/app_runner.ts
+```
+The bosque code in the file can be parsed, type checked, and executed with:
 ```none
-node bin/test/app_runner.js FILE.bsq
+node ref_impl/src/test/app_runner.js FILE.bsq

```

Comment:
Should this be:

```suggestion
node ref_impl/bin/test/app_runner.js FILE.bsq
```

?

Reply:
Thank you) `npm build` does the needed thing.